client/plots/DEanalysis.js

```diff
@@ -32,11 +38,191 @@ class DEanalysis {
 	}
 
 	async main() {
-		const data = await this.app.vocabApi.runDEanalysis(this.state.config)
-		console.log(data)
+	        const data = await this.app.vocabApi.runDEanalysis(this.state.config)	        
+	        console.log(data)
```

Comment:
you could probably remove this console.log if not needed.

Reply:
Done